src/main/java/frc/robot/commands/Score.java

```diff
@@ -9,29 +9,33 @@
 import javax.sql.CommonDataSource;
 
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
 import frc.robot.subsystems.arm.Arm;
 import frc.robot.subsystems.arm.ArmPose;
+import frc.robot.subsystems.arm.Gripper.GripperMode;
 
 // NOTE:  Consider using this command inline, rather than writing a subclass.  For more
 // information, see:
 // https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
 public class Score extends SequentialCommandGroup {
   /** Creates a new Score. */
-  public Score(Arm arm, Supplier<ArmPose> prepPoseSupplier, Supplier<ArmPose> targetPoseSupplier) {
+  public Score(Arm arm, Supplier<ArmPose> prepPoseSupplier, Supplier<ArmPose> targetPoseSupplier, int endWaitSeconds) {
     if (prepPoseSupplier != null){
       addCommands(new MoveArm(arm, prepPoseSupplier));
     }
     Command goToScorePosition = new MoveArm(arm, targetPoseSupplier);
-    Command release = new UnGrip(arm).withTimeout(1);
+    Command release = new InstantCommand(() -> arm.setGripperMode(GripperMode.slowUngrip));
     addCommands(goToScorePosition, release);
     if (prepPoseSupplier != null){
       addCommands(new MoveArm(arm, prepPoseSupplier));
```

Comment:
Why do we do this twice?

Reply:
We want to go to the scoring prep pose, go to the scoring pose, turn on the ungrip, then go back to the prep pose.
This is specifically for cones. For cubes, the prep pose and scoring pose are the same, so it should complete immediately